lib/Scenes/Page/LoginPage/componnent/signup_form.dart

```diff
@@ -12,10 +14,37 @@ class _SignupFormState extends State<SignupForm> {
   final FocusNode _passwordNode = FocusNode();
   final FocusNode _usernameNode = FocusNode();
   final FocusNode _emailNode = FocusNode();
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+  final TextEditingController _usernameController = TextEditingController();
   bool _passwordFocused = false;
   bool _usernameFocused = false;
   bool _emailFocused = false;
   bool _obscureText = true;
+
+  void _HandleSignUpButton() async {
```

Comment:
nama method kecil aja depannya 
```suggestion
  void _handleSignUpButton() async {
```